src/domain/app/footer/FooterCategories.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
old `TopFooter.tsx` with some changes but it seems that git fails to show changed rows after file rename.